src/wallet/spend.cpp

```diff
@@ -187,7 +187,7 @@ CoinsResult AvailableCoins(const CWallet& wallet,
 
             std::unique_ptr<SigningProvider> provider = wallet.GetSolvingProvider(output.scriptPubKey);
 
-            bool solvable = provider ? IsSolvable(*provider, output.scriptPubKey) : false;
+            bool solvable = provider ? IsSolvable(output.scriptPubKey, *provider) : false;
```

Comment:
nit:
```suggestion
            bool solvable = provider && IsSolvable(output.scriptPubKey, *provider)
```

Reply:
Would have taken the suggestion, thanks, but this call site is gone in the rebase. :)